.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
         args: ["--severity=warning"]
 
   - repo: https://github.com/zarhus/oelint-adv
-    rev: 5.5.2-zarhus
+    rev: 5.5.0
```

Comment:
```suggestion
    rev: 5.5.2-zarhus
```

Reply:
Do we have to use zarhus fork still for this linter?

Reply:
Switched to upstream, I see no problems so far.

Our changes seem to be upstreamed already.